src/ray/raylet/scheduling/scheduling_policy.cc

```diff
@@ -40,13 +40,6 @@ int64_t SchedulingPolicy::HybridPolicyWithFilter(
     const ResourceRequest &resource_request, float spread_threshold, bool force_spillback,
     bool require_available, std::function<bool(int64_t)> is_node_available,
     NodeFilter node_filter) {
-  // Step 1: Generate the traversal order. We guarantee that the first node is local, to
-  // encourage local scheduling. The rest of the traversal order should be globally
-  // consistent, to encourage using "warm" workers.
-  std::vector<int64_t> round;
-  round.reserve(nodes_.size());
-  const auto local_it = nodes_.find(local_node_id_);
-  RAY_CHECK(local_it != nodes_.end());
```

Comment:
Is it safe to remove this? Kind of wondering why we have this in the first place lol? It seems a little weird

Reply:
It's not removed. I just moved it to a different place.